docs/src/omnipaxos/round.md

```diff
@@ -0,0 +1,5 @@
+# Round
```

Comment:
I think this section is just too short and confusing. I assume you want to describe how the user should interact the leader election and OmniPaxos... Then provide an example where you get a leader from BLE and call handle_leader in OmniPaxos